RATIS-691. Fix checkstyle violations in ratis-logservice

dineshchitlangia opened the pull request.

joshelser commented on ratis-logservice/src/main/java/org/apache/ratis/logservice/server/LogStateMachine.java:
```diff
@@ -331,7 +331,7 @@ public StateMachineStorage getStateMachineStorage() {
         LogServiceProtos.GetExportInfoReplyProto.newBuilder();
     exportMap.values().stream().map(
-        archivalInfo -> exportBuilder.addInfo(LogServiceProtoUtil.toExportInfoProto(archivalInfo)))
+        archInfo -> exportBuilder.addInfo(LogServiceProtoUtil.toExportInfoProto(archInfo)))
```
What's the checkstyle rule this is failing on?

dineshchitlangia replied:
{{archivalInfo}} is hiding another private member of the class

joshelser commented on ratis-logservice/src/main/java/org/apache/ratis/logservice/server/MetaStateMachine.java:
```diff
@@ -291,7 +290,8 @@ public TransactionContext preAppendTransaction(TransactionContext trx) throws IO
                 MetaServiceProtoUtil.toDeleteLogReplyProto().toByteString()));
     }
 
-//    private CompletableFuture<Message> processCloseLog(MetaServiceProtos.MetaServiceRequestProto logServiceRequestProto) {
```
Please just delete commented out code instead of fixing it.

dineshchitlangia replied:
Sure, I will post a revision shortly.

dineshchitlangia commented:
@joshelser Thanks for taking time to review this PR. There were a few comments from @szetszwo too. I have addressed review comments from both of you in the recent commit.
